src/uu/install/src/install.rs

```diff
@@ -426,18 +426,25 @@ fn copy_files_into_dir(files: &[PathBuf], target_dir: &PathBuf, b: &Behavior) ->
 
     let mut all_successful = true;
     for sourcepath in files.iter() {
-        let targetpath = match sourcepath.as_os_str().to_str() {
-            Some(name) => target_dir.join(name),
-            None => {
-                show_error!(
-                    "cannot stat '{}': No such file or directory",
-                    sourcepath.display()
-                );
+        if !sourcepath.exists() {
+            show_info!(
+                "cannot stat '{}': No such file or directory",
+                sourcepath.display()
```

Comment:
Could you please make sure that the tests are exercising this code path?
thanks!